golang-github-osbuild-composer.spec

```diff
@@ -26,14 +26,15 @@ BuildRequires:  systemd
 BuildRequires:  systemd-rpm-macros
 BuildRequires:  git
 BuildRequires:  golang(github.com/aws/aws-sdk-go)
-BuildRequires:  golang-github-azure-storage-blob-devel
+BuildRequires:  golang(github.com/Azure/azure-storage-blob-go/azblob)
 BuildRequires:  golang(github.com/BurntSushi/toml)
 BuildRequires:  golang(github.com/coreos/go-semver/semver)
 BuildRequires:  golang(github.com/coreos/go-systemd/activation)
 BuildRequires:  golang(github.com/google/uuid)
 BuildRequires:  golang(github.com/julienschmidt/httprouter)
 BuildRequires:  golang(github.com/gobwas/glob)
 BuildRequires:  golang(github.com/google/go-cmp/cmp)
+BuildRequires:  golang(github.com/stretchr/testify/assert)
```

Comment:
How come this wasn't caught in our tests, this is down to modules being enabled before this PR? Will our tests catch stuff like this in the future, once this has been merged?

Reply:
It was only used by test binaries, which were built with `go test` (prior to this patch). `go test` simply downloaded the missing library, because we didn't turn off module support. This patch does that and we should find these kinds of issues in the future.

Probably still better to move to building in `mock`, to be sure.